### DISCUSSION TITLE
MRTK3 with URP in Unity?

### DISCUSSION THREAD

**Discussion by imjusthere** [microsoft/MixedRealityToolkit-Unity#11162] [Q&A] [answered]
I used the MRTK3 demo project and updated MRTK3 to the newest versions to fix some teleportation bugs. Then when I imported the Universal Render Pipeline it broke some of the materials. The controllers are pink. I can't edit the prefab controllers it is using because it tells me the folder is read only. I changed the input folders to not be read only but it doesn't seem to make a difference to Unity. I was able to create my own controller prefabs and fix the textures but I don't see where to replace the original controller prefabs with my own. Any help would be greatly appreciated.

**Comment by Zee2** [comment, marked as answer]
Hi @imjusthere , thanks for reporting this; I've converted this to an issue so we can fix it! See #11165 .

We are definitely supporting URP, I think there's just been an oversight with our material configuration.

**Reply by imjusthere** [reply]
That is terrific. Thank you!
